src/orion/algo/space.py

```diff
@@ -64,7 +64,7 @@ class Dimension(object):
 
     """
 
-    NO_DEFAULT_VALUE = object()
+    NO_DEFAULT_VALUE = None
```

Comment:
wow. It seems that there is something particular about `object()`. How can it be recreated with each import, isn't it a class variable?

Reply:
I might have been confused when I wrote the commit message. I believe the main issue was about serialization.